cob_bringup/package.xml

```diff
@@ -80,18 +72,14 @@
   <exec_depend>openni_launch</exec_depend>
   <exec_depend>openni2_launch</exec_depend>
   <exec_depend>position_controllers</exec_depend>
```

Comment:
this is the same as costmap_2d...cob_hardware_config specifies it as a controller type...so in order to load it, we need to specify the dependency....